be/src/exec/csv_scanner.cpp

```diff
@@ -48,8 +48,8 @@ static std::string string_2_asc(const std::string& input) {
 static std::string make_column_count_not_matched_error_message(int expected_count, int actual_count,
                                                                CSVParseOptions& parse_options) {
     std::stringstream error_msg;
-    error_msg << "Value count does not match column count: "
-              << "expected = " << expected_count << ", actual = " << actual_count << ". "
+    error_msg << "Target column count: " << expected_count
+              << " doesn't match value column count in file: " << actual_count << ". "
```

Comment:
the source of stream load is not a file?

Reply:
fixed